src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -1151,15 +1151,25 @@ public function originalIsEquivalent($key)
 
         if ($attribute === $original) {
             return true;
-        } elseif (is_null($attribute)) {
-            return false;
```

Comment:
What happened to the this is_null early return here?

Reply:
I'm not convinced that it *adds* much. Following the method flow it will return false in the end.